book/accessibility.md

```diff
@@ -106,7 +115,7 @@ In the United States, the European Union, and many other countries,
 website accessibility is legally required. For example, United States
 Government websites are required to be accessible under [Section
 508][sec508] of the [Rehabilitation Act Amendments of
-1993][rehab-act], and the government has a bunch of
+1973][rehab-act], and the government has a bunch of
```

Comment:
I think, confusingly, these are two similarly-named acts. We should double-triple check.

Reply:
I looked at the site you recommended, and it's the ...act of 1973, with amendments made in 1998. I added a parenthetical.